consistency-checker/src/main/java/com/redhat/lightblue/migrator/consistency/MigrationJob.java

```diff
@@ -367,29 +370,77 @@ private int overwriteLightblue(List<JsonNode> documentsToOverwrite) {
     }
 
     protected boolean documentsConsistent(JsonNode sourceDocument, JsonNode destinationDocument) {
-        boolean consistent = true;
+        return doDocumentsConsistent(sourceDocument, destinationDocument, null);
+    }
 
-        Iterator<Entry<String, JsonNode>> nodeIterator = sourceDocument.fields();
+    //Recursive method
+    private boolean doDocumentsConsistent(final JsonNode sourceDocument, final JsonNode destinationDocument, final String path) {
+        List<String> excludes = getJobConfiguration().getComparisonExclusionPaths();
+        if(excludes != null && excludes.contains(path)) {
```

Comment:
For the first run, the path will be null by design. I am handling the null case further down on line 432. StringUtils.isEmpty(path) will check for null or an empty string.